client/payment-details/summary/index.tsx

```diff
@@ -3,11 +3,12 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { sprintf, __ } from '@wordpress/i18n';
```

Comment:
There are unused imported symbols in this file, can we clean them up please?

Reply:
Sorry about missing this. I have cleaned it up here d023fe1